aws_protocol_adaptor/device_client/aws_nvmsgbroker.c

```diff
@@ -33,6 +33,7 @@
 #include "aws_config_parser.h"
 #include "nvds_msgapi.h"
 #include "aws_nvmsgbroker.h"
+#define MAX_SUBSCRIPTIONS 50
```

Comment:
I think the `aws_iot_config.h` file has a bunch of similar definitions in it. This probably belongs there.